students/wachulec_maria/lesson_02_flow_control/the_second_maximum.py

```diff
@@ -0,0 +1,15 @@
+base_1 = int(input("Number: "))
+base_2 = int(input("Number: "))
```

Comment:
Little bit too foggy var names. it's better to aim in some clear var names like `second_max, first_max`